spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -42,14 +42,17 @@ A *bidirectional* channel is a channel where packets can flow in both directions
 
 A *unidirectional* channel is a channel where packets can only flow in one direction: from `A` to `B` (or from `B` to `A`, the order of naming is arbitrary).
 
-An *ordered* channel is a channel where packets are delivered exactly in the order which they were sent.
+An *ordered* channel is a channel where packets are delivered exactly in the order which they were sent. This channel type offers a very strict guarantee of ordering. Either, the packets are received in the order they were sent, or if a packet in the sequence times out; then all future packets are also not receivable and the channel closes.
```

Comment:
This is the current semantics of Ordered Channels? I did not know that.

That definitely makes sense to add a version with looser requirements.

(This will require a hard fork on both sides to add it?)

Reply:
Correct it will require both sides to have the new type enabled